src/gluonnlp/data/sampler.py

```diff
@@ -504,6 +504,8 @@ class SplitSampler(Sampler):
     """Split the dataset into `num_parts` parts and randomly sample from the part
     with index `part_index`.
 
+    The data are randomly shuffled at each iteration within each partition.
```

Comment:
are -> is ?

Reply:
data is plural. datum is singular

Reply:
Citing "https://www.quickanddirtytips.com/education/grammar/data-is-data-are"

```
‘Data’: The Latin Plural of ‘Datum”
The word “data” comes to English from Latin, in which “datum” is the singular and “data” is the plural. If you’re sticking with that history, it should be “Your data are now safe.” “Data.” Plural.

‘Data’: The English Singular Meaning ‘Information’
But I bet that sounds weird to a lot of you because since the 1940s, people have been using “data” as a singular noun more and more often, especially in general writing.
```

But anyway, it's just a matter of convention.

Reply:
Updated. I am catching up with the world!

Reply:
😸